operational/vmware/instance_tag_sync/instance_tag_sync.pt

```diff
@@ -117,14 +117,21 @@ end
 policy "policy_instance_sync" do
   validate_each $ds_instances_and_tags do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} tags will be synced on {{ parameters.param_cloud_name }}"
-    detail_template <<-EOS
-# tags synced on {{ parameters.param_cloud_name }}
-| instanceId | name | state | tags |
-| -------- | ------ | ------- | -------- |
-{{ range data -}}
-  | {{.id}} | {{.name}} | {{.state}} | {{ range $i, $e := .tags }}{{ if $i }}, {{ end }}{{ $e }}{{ end }} |
-{{ end -}}
-EOS
+    export do
+      resource_level true
+      field "id" do
```

Comment:
Id should be the last column
add cloud as the first column and sorted.

Reply:
added cloud.